examples/python/standalone/weaviate-semantic-search.py

```diff
@@ -0,0 +1,134 @@
+# PREREQUISITE:
+#   Start up Weaviate locally as described on the [Docker Compose](https://weaviate.io/developers/weaviate/installation/docker-compose)
+#   installation guide.
+# DEPENDENCIES:
+#   `pip install -U weaviate_client sentence_transformers ironcore-alloy`
```

Comment:
I think we need to pin versions here. I didn't recognize this in the PRs for opensearch and elasticsearch, but the likeyhood that this will keep working long term is low if we don't pin. 

I'd suggest moving this into its own subdirectory and pinning versions using `requirements.txt`.